For some reason this was set as a noinst library, but it definitely
needs to be installed.

Index: openhpi-2.14.1/src/Makefile.am
===================================================================
--- openhpi-2.14.1.orig/src/Makefile.am	2009-11-23 15:18:36.000000000 -0500
+++ openhpi-2.14.1/src/Makefile.am	2011-03-04 10:58:06.520681389 -0500
@@ -41,7 +41,7 @@
 
 SUBDIRS                 = t
 
-noinst_LTLIBRARIES     = libopenhpi.la
+lib_LTLIBRARIES         = libopenhpi.la
 
 libopenhpi_la_SOURCES   = \
                            config.c \
